### PR TITLE
Prevent MySQL disk full

### PR DESCRIPTION
**Pull request by laysauchoa**
Migration of: https://help.aiven.io/en/articles/3781502-mysql-disk-full-prevention

Tested: 
Screenshot 2022-04-22 at 15 28 19

Fixes: https://github.com/aiven/devportal/issues/666

### CODE REVIEW

**Comment by laysauchoa** [comment]
Would be possible someone to review this? thank you

File: docs/products/mysql/howto/prevent-disk-full.rst [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+Actions to make more space available
+------------------------------------
+
+Identify and optimize problem tables
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+InnoDB does not reclaim unused disk space by default and this can cause a disk to become full. 
+
+Read the help article MySQL disk usage for more information.
```
Comment (TibsAtWork) [inline docs/products/mysql/howto/prevent-disk-full.rst:21]: Missing link for `MySQL disk usage`

Presumably still https://help.aiven.io/en/articles/4808068-mysql-disk-usage, unless it's been ported.

**Review by TibsAtWork — COMMENTED** [review]
Looks good to me, apart from the missing link. Once that's fixed I'm happy to approve.

File: docs/products/mysql/howto/prevent-disk-full.rst (hunk 1 above)
Comment (laysauchoa) [reply docs/products/mysql/howto/prevent-disk-full.rst]: thanks, fixed!

**Review by TibsAtWork — APPROVED** [review]
Thanks

**Pull request merged**